src/SelfServe/SelfServeTypes.ts

```diff
@@ -113,18 +131,23 @@ export interface SmartUiInput {
   disabled?: boolean;
 }
 
-export enum SelfServeNotificationType {
-  info = "info",
-  warning = "warning",
-  error = "error",
-}
-
-export interface SelfServeNotification {
-  message: string;
-  type: SelfServeNotificationType;
+export interface OnSaveResult {
+  operationStatusUrl: string;
+  requestInitializedPortalNotification: {
+    titleTKey: string;
+    messageTKey: string;
+  };
+  requestCompletedPortalNotification: {
```

Comment:
>requestCompletedPortalNotification [](start = 2, length = 34)

need to take care of scenario where there is failure after polling the operation status url